test_sets/protocol_nak_reason_param.txt

```diff
@@ -1,4 +1,14 @@
 ./test_scripts/Protocol/NACKReason/001_Second_Protected_Video_Service_NACK.lua
 ./test_scripts/Protocol/NACKReason/002_Second_Protected_Audio_Service_NACK.lua
 ./test_scripts/Protocol/NACKReason/003_Second_Protected_RPC_Service_NACK.lua
-./test_scripts/Protocol/NACKReason/004_Invalid_Certificate_NACK.lua
\ No newline at end of file
+./test_scripts/Protocol/NACKReason/004_Invalid_Certificate_NACK.lua
+./test_scripts/Protocol/NACKReason/005_Wrong_app_type_Video_Audio_Services_NACK.lua
+./test_scripts/Protocol/NACKReason/006_Wrong_HMI_level_Video_Audio_Services_NACK.lua
+./test_scripts/Protocol/NACKReason/007_Force_protected_Video_Audio_Services_NACK.lua
+./test_scripts/Protocol/NACKReason/008_Force_not_protected_Video_Audio_Services_NACK.lua
+./test_scripts/Protocol/NACKReason/009_PTU_failed_Video_Service_NACK.lua
+./test_scripts/Protocol/NACKReason/010_PTU_failed_Audio_Service_NACK.lua
+./test_scripts/Protocol/NACKReason/011_PTU_failed_RPC_Service_NACK.lua
+./test_scripts/Protocol/NACKReason/012_Invalid_time_Video_Service_NACK.lua
+./test_scripts/Protocol/NACKReason/013_Invalid_time_Audio_Service_NACK.lua
+./test_scripts/Protocol/NACKReason/014_Invalid_time_RPC_Service_NACK.lua
```

Comment:
I would propose to add scripts In case mobile app starts (not protected) AUDIO/VIDEO service  two times